src/lime/_internal/backend/native/NativeCFFI.hx

```diff
@@ -413,8 +415,7 @@ class NativeCFFI
 		"lime_file_watcher_add_directory", "oobo", false));
 	private static var lime_file_watcher_remove_directory = new cpp.Callable<cpp.Object->cpp.Object->cpp.Void>(cpp.Prime._loadPrime("lime",
 		"lime_file_watcher_remove_directory", "oov", false));
-	private static var lime_file_watcher_update = new cpp.Callable<cpp.Object->cpp.Void>(cpp.Prime._loadPrime("lime", "lime_file_watcher_update", "ov",
-		false));
```

Comment:
These are a lot of changes unrelated to the pull request. How about splitting them into a separate one?

Reply:
This was an autoformatting thing. I might manually redo this file based off the main branch to undo these.